src/swift/client-ios/Sessionless/Sessionless/Classes/Sessionless.swift

```diff
@@ -11,7 +11,7 @@ import JavaScriptCore
 public class Sessionless {
     public struct Keys {
         public let publicKey: String
-        public let privateKey: String
+        let privateKey: String
```

Comment:
I am not to sure about this, but should we allow users to create Sessionless objects with predefined private keys? I've implemented Sessionless in one of my commercial projects where one server has to communicate with another. Each server has 2 values saved. Its own private key and the other server's public key. If one of the servers generates a new private key, the public key becomes obsolete which renders the communication impossible until the new public keys is acknowledged by the other server. With the predefined private keys it's possible to re-run the servers without the need for key exchange.

Reply:
right right right, I thought there was a reason we did it this way, and then forgot when I was updating the docs. I'll change this back.